demisto_sdk/commands/common/tools.py

```diff
@@ -190,8 +190,10 @@ def get_log_verbose() -> bool:
     return LOG_VERBOSE
 
 
-def get_mp_tag_parser():
+def get_mp_tag_parser(marketplace: Optional[MarketplaceVersions] = None):
     global MARKETPLACE_TAG_PARSER
+    if marketplace:
+        return marketplace
```

Comment:
Please remove this change.
This behavior is only being used by one function and makes little sense.